engine/crates/federated-graph/src/render_sdl/render_federated_sdl.rs

```diff
@@ -203,7 +207,12 @@ pub fn render_federated_sdl(graph: &FederatedGraph) -> Result<String, fmt::Error
         let mut members = union.members.iter().peekable();
 
         while let Some(member) = members.next() {
-            sdl.push_str(&graph[graph[*member].name]);
+            sdl.push_str(
+                graph
+                    .through(*member)
+                    .through(|member| member.type_definition_id)
+                    .str(|def| def.name),
```

Comment:
We could make this a bit nicer to read with one of the derive crates for getters:

```rust
graph
	.through(*member)
        .through(Object::type_definition_id)
        .str(TypeDefinition::name)
```